docs/_main/getting_started.md

```diff
@@ -22,23 +22,19 @@ status. Before we can call any methods, we need to configure our new app with th
 ## Getting a token to use the Web API
 
 Navigate to "OAuth & Permissions" and scroll down to the section for scopes. Slack describes the various permissions
-your app could obtain from an installing user as **scopes**. There are [over 60 scopes](https://api.slack.com/scopes)!
+your app could obtain from an installing bot as **scopes**. There are [over 80 scopes](https://api.slack.com/scopes)!
 Some are broad and authorize your app to access lots of data, while others are very specific and let your app touch just
 a tiny sliver. Your users (and their IT admins) will have opinions about which data your app should access, and only
 agree to install the app if the data permissions seem reasonable, so we recommend finding the scope(s) with the least
 amount of privilege for your app's needs. In this guide we will use the Web API to post a message. The scope required
-for this is called `chat:write:user`. Use the dropdown or start typing its name to select and add the scope, then click
+for this is called `chat:write`. Use the dropdown or start typing its name to select and add the scope, then click
```

Comment:
For GBP, there are two sections of scopes. Clarify that they should use the dropdown under the "Bot Token Scopes" header